content/geometry/object-human/human-structure.adoc

```diff
@@ -0,0 +1,861 @@
+            |_ [[../geometry/object-classes.html#_full_toes_left Full_Toes_Left]]
+      |_ [[../geometry/object-classes.html#_upper_leg_right Upper_Leg_Right]]
+        |_ [[../geometry/object-classes.html#_lower_leg_right Lower_Leg_Right]]
+          |_ [[../geometry/object-classes.html#_foot_right Foot_Right]]
+            |_ [[../geometry/object-classes.html#_full_toes_right Full_Toes_Right]]
+  |_ [[../geometry/object-classes.html#_accessories Accessories_Wristband]]
+  |_ [[../geometry/object-classes.html#_body Body_Human]]
+  |_ [[../geometry/object-classes.html#_clothing Clothing_Vest]]
+  |_ [[../geometry/object-classes.html#_hair Hair_Short]]
+----
+
+== Grp_Root 
+
+This group is used as a parent for all following nodes. It can be used to place the whole asset and select the complete node hierarchy at once. The transformation is set to 0 for all axes.
+
+.Grp_Root 
+[%header, cols="20, 80"]
+|===
+
+2+^| Grp_Exterior
+
+| *Origin*
+| Center of the bounding box on the ground
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing sidewards
+
+| *z-axis*
+| Pointing upwards
+|===
+
+
+== Accessories_<Name>
+
+This object represents an additional or exchangeable accessory of the human.
+
+.Accessories 
+[%header, cols="20, 80"]
+|===
+
+2+^| Accessories
+
+| *Origin*
+| Center of the bounding box on the ground
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing sidewards
+
+| *z-axis*
+| Pointing upwards
+
+|===
+
+
+== Body_<Name>
+
+This object represents the body of the human.
+
+.Body
+[%header, cols="20, 80"]
+|===
+
+2+^| Body
+
+| *Origin*
+| Center of the bounding box on the ground
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing sidewards
+
+| *z-axis*
+| Pointing upwards
+|===
+
+
+== Clothing_<Name>
+
+This object represents an additional or exchangeable clothing part of the human.
+
+.Clothing
+[%header, cols="20, 80"]
+|===
+
+2+^| Clothing
+
+| *Origin*
+| Center of the bounding box on the ground
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing sidewards
+
+| *z-axis*
+| Pointing upwards
+|===
+
+
+== Hair_<Name>
+
+This object represents an additional or exchangeable hair part of the human.
+
+.Hair
+[%header, cols="20, 80"]
+|===
+
+2+^| Hair
+
+| *Origin*
+| Center of the bounding box on the ground
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing sidewards
+
+| *z-axis*
+| Pointing upwards
+|===
+
+
+== Root
+
+The `Root` bone is the parent bone for all other bones. It can be used to control the whole skeleton.
+
+.Root bone
+[%header, cols="20, 80"]
+|===
+
+2+^| Root
+
+| *Origin*
+| World coordinate system
+
+| *x-axis*
+| Pointing forwards
+
+| *y-axis*
+| Pointing sidewards
+
+| *z-axis*
+| Pointing upwards
+|===
+
+NOTE: Please add a figure and description.
+
+== Lower_Spine
+
+The `Lower_Spine` bone represents the lowest parts and bones of the spine, that is, the Coccyx and Sacrum spine bones of the human skeleton.
```

Comment:
@ClemensLinnhoff please check whether this definition structure works for you :)